source/stylesheets/corporate/sections/_team.sass

```diff
@@ -4,8 +4,12 @@
 
 #team
   padding-bottom: $sectionsMargin
-  .col:hover .avatar
-    @include filter(none)
+  .col
+    &:hover .avatar
+      @include filter(none)
+    &:nth-child(4)
+      margin-left: 0
+      margin-right: 0
```

Comment:
Having worked on bootstrap project just before I mixed up things.
My coworker made same comment, and I'm happy to see it's flexbox.

We applied those changes instead:

dd0599ab7ffd1ba230a24131927302b0fbfeb249
9bb0767c2faa000a58abf1cb11cf9d675e4bd8f6

Last one to improve the justification of wrapped items when less than 3.

Thanks for the comment!